tests/unit/utils/test_usage_tracking.py

```diff
@@ -112,4 +112,4 @@ def test_device_id_persisted():
             assert f.read() == new_dev_id
     finally:
         # restore the old devid back after the test
-        shutil.move(old_devid_path, devid_path)
+        shutil.move(str(old_devid_path), devid_path)
```

Comment:
This was necessary to pass mypy pre-commit hook.

Reply:
see above

Reply:
ack